java/ql/src/change-notes/2022-08-01-android-debug-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* Added a new query, `java/android/debuggable-attribute-enabled`, to detect if the `android:debuggable` attribute is enabled.
```

Comment:
```suggestion
* Added a new query, `java/android/debuggable-attribute-enabled`, to detect if the `android:debuggable` attribute is enabled in the Android manifest.
```

Reply:
Fixed in commit [a0cf7c9](https://github.com/github/codeql/pull/9939/commits/a0cf7c9f1b1f53de19b8c8333991aaf8d89342b5).